massa-final-state/src/test_exports/config.rs

```diff
@@ -2,26 +2,42 @@
 
 //! This file defines testing tools related to the configuration
 
-use std::collections::BTreeMap;
+use crate::{FinalState, FinalStateConfig};
+use massa_async_pool::{AsyncPool, AsyncPoolConfig};
+use massa_ledger::{FinalLedger, LedgerConfig};
+use massa_models::Slot;
 
-use crate::FinalStateConfig;
-use massa_async_pool::AsyncPoolConfig;
-use massa_ledger::LedgerConfig;
-use tempfile::{NamedTempFile, TempDir};
+/// Default value of `FinalState` used for tests
+impl Default for FinalState {
+    fn default() -> Self {
+        let config = FinalStateConfig::default();
+        let slot = Slot::new(0, config.thread_count.saturating_sub(1));
 
-impl FinalStateConfig {
-    /// Get sample final state configuration
-    pub fn sample() -> (Self, NamedTempFile, TempDir) {
-        let (ledger_config, keep_file, keep_dir) = LedgerConfig::sample(&BTreeMap::new());
```

Comment:
You've removed the `LedgerConfig::sample(&BTreeMap::new());` call, this is needed by the bootstrap test, it will panic with a default config.